.github/workflows/discord_notify.yml

```diff
@@ -0,0 +1,22 @@
+name: Discord Notify
+
+on:
+  pull_request:
+    types: [ labeled ]
```

Comment:
There's some delay between the PR being opened and butterfly labeling it, maybe using the [label created event](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#label) might be less flaky?

Reply:
Oops, https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target